src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -217,6 +217,12 @@ public boolean isExternalSubsParsed() {
 	private byte[] h264_annexB;
 
 	/**
+	 * @deprecated Use standard getter and setter to access this variable.
+	 */
+	@Deprecated
+	public String truncated;
+
+	/**
 	 * Not stored in database.
```

Comment:
@Sami32 Hehe, I know what you mean. But my experience is that if you take the time to learn a few of the basic concepts for a given topic, most of what you read about it becomes much more logical afterwards.

I don't know if that Wikipedia article was good or bad, as I can't read French - but it is possible to explain the basic concepts without making it too difficult. If you look around a little, I'm sure you can find [something that's understandable](http://jmdoudoux.developpez.com/cours/developpons/java/chap-poo.php).